app/src/scripts/game/components/header.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import {connect} from 'react-redux';
-import { questUnlocked, addComplete } from '../../_utils';
+import { questUnlocked, addComplete, getAge, agesData } from '../../_utils';
```

Comment:
Again, agesData would be a `private` variable, only used bt the utils function directly. So I'd not export that here.

```suggestion
import { questUnlocked, addComplete, getAge } from '../../_utils';
```